hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconContainerManager.java

```diff
@@ -258,6 +262,26 @@ public void addNewContainer(ContainerWithPipeline containerWithPipeline)
     }
   }
 
+  /**
+   * Add a new pipeline to the pipeline metadata.
+   *
+   * @param pipeline pipeline
+   * @throws IOException
+   */
+  public void addPipeline(Pipeline pipeline)
```

Comment:
@ArafatKhan2198 thanks for working on this patch. You don't need to add this new method here, You can directly call `addPipeline` method on existing `pipeLineManager` instance above.

Reply:
Thanks done!